src/lib/search/spells.ts

```diff
@@ -22,18 +22,22 @@ export function buildSpellIndex() {
     return flexIndex;
 }
 
-export function searchSpells(query: string, limit: number = 5, offset: number = 0): Spell[] {
+export function searchSpells(query: string, limit: number = 5, offset: number = 0, a5e: boolean = false): Spell[] {
     if (!query.length) {
         return spells.slice(offset, offset + limit);
     }
 
     const options: FlexSearch.SearchOptions = {
         limit: limit,
         offset: offset,
-        suggest: true,
+        suggest: false,
     };
 
     let results = spellIndex.search(query, options);
 
-    return results.map(r => spells[r as number]);
+    if (a5e) {
+        return results.map(r => spells[r as number]);
+    }
+    const allSpellsResult: Spell[] = results.map(r => spells[r as number]);
+    return allSpellsResult.filter(spell => !spell.slug.includes("-a5e"));
```

Comment:
Kan gøres simplere ved at invertere `if (a5e)` blokken.

```suggestion
    let results = spellIndex.search(query, options).map(r => spells[r as number]);

    if (!a5e) {
        results.filter(spell => !spell.slug.includes("-a5e"));
    }

    return results;
```